mcp_zammad/models.py

```
@@ -57,6 +59,32 @@ class ArticleSender(str, Enum):
+        """Sanitize filename to prevent path traversal."""
+        # Remove path components, keep only basename, and remove null bytes
+        return os.path.basename(v).replace("\x00", "")
+
+    @field_validator("data")
+    @classmethod
+    def validate_base64(cls, v: str) -> str:
+        """Validate base64 encoding."""
+        try:
+            base64.b64decode(v, validate=True)
+        except Exception as e:
+            raise ValueError(f"Invalid base64 encoding: {e}") from e
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)